### DISCUSSION TITLE
Show online statistics on results screen

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#17369] [Ideas] [answered]
I am kinda missing this feature from stable. I always liked seeing how a score influenced my online stats like score and pp.

![onlinestats](https://user-images.githubusercontent.com/22381687/159170743-64fdf53f-408f-48c2-8c93-392a88a0a6b6.png)
On stable you can see this when scrolling down on the results screen. Not sure how this could be implemented in lazer. There is the functionality to click on the results card to get more info about the score, but I'm not sure it would really fit in there.

**Comment by Walavouchey** [comment]
Scores in lazer aren't shown in beatmap leaderboards yet, so this would come after that

**Reply by respektive** [reply]
Yeah that's true, but some stats already update like total score. Also i feel like its probably best to have this in place once the scores actually submit to leaderboards. :thinking:
For the time being it could just show no updates for pp and such.

**Comment by peppy** [comment]
It will of course return. Currently this is a hole in @arflyte's provided designs. Has been brought up before but nothing locked in yet.

**Comment by arflyte** [comment]
Hmm, I don't have anything for this yet. Will add this to the new design.

**Reply by respektive** [reply]
About the new design, is it shared publicly anywhere? It's not in the [figma library](https://www.figma.com/file/6m10GiGEncVFWmgOoSyakH/osu!-Figma-Library).

**Reply by peppy** [reply]
it's not done yet. also please only request designs if you are planning on implementing them.

**Comment by respektive** [comment, marked as answer]
https://github.com/ppy/osu/issues/18877
